scripts/solve_network.py

```diff
@@ -841,6 +876,26 @@ def solve_network(n, config, solving, opts="", **kwargs):
     if "infeasible" in condition:
         raise RuntimeError("Solving status 'infeasible'")
 
+    # check if enhanced_geothermal_performant might have changed model results
```

Comment:
I would remove this check for the main branch.

Reply:
Just because its too many lines in a quite prominent part of the code for little effect? Would be fair enough

Reply:
removed it :)